src/pages/tasks/NewTaskDescriptionPage.js

```diff
@@ -36,6 +36,8 @@ const defaultProps = {
 };
 
 const NewTaskDescriptionPage = (props) => {
+    const descriptionRef = useRef(null);
```

Comment:
Change to `descriptionInputRef` or to simply `inputRef` to be consist with other pages (same note for `titleRef`)